app/Transformers/BillTransformer.php

```diff
@@ -263,11 +264,14 @@ protected function payDates(Bill $bill): array
         }
         $set          = new Collection();
         $currentStart = clone $this->parameters->get('start');
-        // 2023-06-23 subDay to fix 7655
-        $currentStart->subDay();
```

Comment:
Why is this removed?

Reply:
well, the linked issue is the main reason for this PR and the changes done no longer have a requirement for this "->subDay()".